src/node.js

```diff
@@ -630,6 +630,8 @@
               writable: false
             });
           }
+          // Ensure that Streams2 won't try to `.end()` the stream
```

Comment:
nit: might rephrase this comment to `make sure that stdin can't be .end()-ed`, since we're on streams3 now.